### PR TITLE
Team manager

### PR DESCRIPTION
**Pull request by elyhess**
Add team manager and team classes.
All tests passing.

### CODE REVIEW

File: lib/team_manager.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  attr_reader :teams_data, :teams
+
+  def initialize(file_locations)
+    @teams_data = file_locations[:teams]
+  end
+
+  def all
+    teams = []
+    CSV.foreach(@teams_data, header_converters: :symbol, headers: true) do |row|
+      team_data = {
```
Comment (philmccarthy) [inline lib/team_manager.rb:13]: should we standardize the variable name here to match our Game class, `team_attributes` ?

File: test/team_test.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/team'
+
+class TeamTest < Minitest::Test
+
+  def setup
+    @team_info = {:team_id=>"1",
```
Comment (philmccarthy) [inline test/team_test.rb:8]: here's where we actually don't need an instance variable for `team_attributes`

**Review by philmccarthy — CHANGES_REQUESTED** [review]
Minor updates in my review, check the comments. Thanks & nice work so far!

**Comment by kaiheiongaku** [comment]
Looks good!

**Pull request merged**